### DISCUSSION TITLE
如何选择 TeX Live 版本？

### DISCUSSION THREAD

**Discussion by zhuohoudeputao** [mengchaoheng/SCUT_thesis#11] [General]
目前我使用的系统是Ubuntu 20.04，TeX Live 版本是 2019，编译一切正常。但是在Win11上使用TeX Live 2021编译就一直会出现一个`missing $`的问题。不知道是否是版本问题。

**Comment by mengchaoheng** [comment]
猜到了是缺少字体的问题，不过在ubuntu上写太厉害了，是物理机开发环境不想切换电脑吗？
